api/src/main/resources/workbench.yaml

```diff
@@ -1202,6 +1203,27 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
+  /v1/cron/billingProjectGarbageCollection:
+    get:
+      security: []
+      tags:
+        - offlineBilling
+        - cron
+      description: >
+        Endpoint meant to be called offline to trigger a bulk transfer of ownership of billing
```

Comment:
nit: offline usage is implied from the `offlineBilling` tag and cron usage from `cron`